books/free-programming-books-langs.md

```diff
@@ -1390,6 +1390,12 @@ Books on general-purpose programming that don't focus on a specific language are
 * [The Svelte Handbook](https://flaviocopes.com/page/svelte-handbook/) - Flavio Copes (PDF, EPUB, Kindle) *(email address requested)*
 
 
+#### SvelteKit
```

Comment:
Don't think this should be a new language, it's just Svelte

Reply:
Well because there is nuxt and next.js, then i thought SvelteKit should be on new section ? 
it was the same meta-framework that based on svelte (with nuxt for vue and next for react).

Reply:
The separate category for nuxt is probably a mistake. I don't know enough about next to judge. Will raise these as an issue. It depends a lot on the number of resources. It's probably easier to put them all in svelte.

The categories exist to help people find things. no one will miss a sveltekit resource if it's mixed with svelte-only resources. And have you looked at the svelte resources to make sure they aren't about sveltekit?